### PR TITLE
Merge Return Results

### PR DESCRIPTION
**Pull request by SpookyBeverage**
Finished up Return Results should be good to merge now,
Also deleted a lot of commented code that I ran upon

### CODE REVIEW

File: apps/frontend/components/Dashboard.js [hunk 1]
```diff
@@ -190,43 +177,30 @@ const Navbar = (props) => {
 
 
 const ExpLog = ({projectinit, uid}) => {
-    
     const [project, setProject] = useState(projectinit);
-    subscribeToExp(project.id, uid, setProject)
+    // subscribeToExp(project.id, uid, setProject)
```
Comment (budak7273) [inline apps/frontend/components/Dashboard.js:181]:
```suggestion
```

File: apps/frontend/components/Dashboard.js [hunk 2]
```diff
@@ -286,7 +260,12 @@ const SearchBar = (props) => {
 export default function Dashboard({ user, experimentss }) {
     const [experiments, setExperiments] = useState(experimentss);
 
-    listenToNew((payload)=> setExperiments([...experiments, payload]))
+    // listenToNew((payload)=> setExperiments([...experiments, payload]))
```
Comment (budak7273) [inline apps/frontend/components/Dashboard.js:263]:
```suggestion
```

File: apps/frontend/components/NewExp.js [hunk 3]
```diff
@@ -165,7 +162,7 @@ const ConfirmationStep = ({ form, ...props }) => {
 	);
 };
 const dropzoneKids = (status) => {
-    console.log(status);
+    // console.log(status);
```
Comment (budak7273) [inline apps/frontend/components/NewExp.js:165]:
```suggestion
```

**Review by rrobinson281 — DISMISSED** [review]
Maybe avoid deleting commented code preemptively please, otherwise good job

File: apps/frontend/pages/dashboard.js [hunk 4]
```diff
@@ -10,16 +31,22 @@ export async function getServerSideProps({ req }) {
...
-	if (error) {
-		console.log(error);
-	}
+	// const { data, error } = await supabase
+	// 	.from('experiments')
+	// 	.select('*')
+	// 	.eq('creator', user.id);
+	// if (error) {
+	// 	console.log(error);
+	// }
```
Comment (budak7273) [inline apps/frontend/pages/dashboard.js:40]:
```suggestion
```

**Review by budak7273 — CHANGES_REQUESTED** [review]
No commented code without explanation comments please.

There is stuff like missing semicolons but I'm not going to nag on that because the linter will fix it later once dev containers are ready.

I can't test this locally yet so it would be great if @rrobinson281 or @rhit-chanb could test functionality

**Review by budak7273 — APPROVED** [review]

**Pull request merged**